src/MonoMod.FrameworkTests/Program.cs

```diff
@@ -1,20 +1,13 @@
 ﻿#define MM
 
-using Mono.Cecil.Cil;
-using MonoMod.Cil;
-using MonoMod.Core;
-using MonoMod.Core.Platforms;
 using MonoMod.RuntimeDetour;
 
-using MonoMod.Backports;
 using System;
 using System.Diagnostics;
-using System.Runtime.CompilerServices;
 using MonoMod.Utils;
 using System.Runtime.InteropServices;
 
 #if NETCOREAPP1_0_OR_GREATER
-using Xunit;
 using Xunit.Abstractions;
 #endif
```

Comment:
This is in a similar situation to the MacOS system implementation; all of these have been used at some point and will likely be used in the future. FrameworkTests is a project used exclusively for experimenting and testing different aspects of MonoMod on various platforms without dealing with the stuff Xunit runs before actual tests.